fed/mosaic-sns/src/main/java/org/eclipse/mosaic/fed/sns/model/SimpleAdhocTransmissionModel.java

```diff
@@ -46,18 +46,18 @@ public class SimpleAdhocTransmissionModel extends AdhocTransmissionModel {
      */
     public CTransmission simpleMultihopTransmission = new CTransmission();
 
+
     /**
-     * Simulates a direct transmission between the sender and receivers. Note: If a single addressed receiver is used
-     * the 'receivers' map will only contain one entry as well as the results.
+     * Simulates a direct transmission between the sender and receivers.
      *
      * @param senderName            The sender of the transmission.
-     * @param receivers             The receivers of the transmission.
+     * @param receivers             The receivers of the unicast transmission.
      * @param transmissionParameter Data class holding the maximumTtl, the {@link Delay} and the current map of simulated entities
      * @param currentNodes          a reference to all currently online nodes
-     * @return Map of the receivers and their transmission results.
+     * @return The transmission result to the single receiver.
      */
     @Override
-    public Map<String, TransmissionResult> simulateTopocast(
+    public Map<String, TransmissionResult> simulateSinglehop(
             String senderName, Map<String, SimulationNode> receivers,
             TransmissionParameter transmissionParameter, Map<String, SimulationNode> currentNodes) {
         return calculateTransmissions(
```

Comment:
Now the name `calculateTransmissions` is confusing to me. 

I propose `simulateTransmissions` or `simulateMultipleTransmissions` 

Maybe better the second one to distinguish it well from `simulateTransmission` (singular)